sdk/state/state.go

```diff
@@ -22,7 +22,6 @@ type Amount struct {
 type EscrowAccount struct {
 	Address        *keypair.FromAddress
 	SequenceNumber int64
-	Balances       []Amount
```

Comment:
not being used and we don't need this right now. When we add support for multiple assets we might add back